azure-pipelines.yml

```diff
@@ -192,38 +244,53 @@ stages:
           projectName: 'SonarAnalyzer .Net'
           projectVersion: '$(SONAR_PROJECT_VERSION)'
           extraProperties: |
-            sonar.cs.opencover.reportsPaths="$(Build.SourcesDirectory)/**/coverage.**.xml"
-            sonar.cs.vstest.reportsPaths="$(Build.SourcesDirectory)/**/tests/SonarAnalyzer.UnitTest/TestResults/*.trx"
+            sonar.cs.opencover.reportsPaths="$(Build.SourcesDirectory)/coverage/**.xml"
+            sonar.cs.vstest.reportsPaths="$(Build.SourcesDirectory)/TestResults/*.trx"
             sonar.analysis.buildNumber=$(Build.BuildId)
             sonar.analysis.pipeline=$(Build.BuildId)
             sonar.analysis.sha1=$(Build.SourceVersion)
             sonar.analysis.repository=$(Build.Repository.ID)
 
-      - powershell: |
-          . .\scripts\utils.ps1
-          cd analyzers
-
-          nuget restore -LockedMode -ConfigFile "NuGet.Config" SonarAnalyzer.sln
+      - task: VSBuild@1
+        displayName: 'Set BranchName, Sha1 and BuildNumber properties from Azure pipeline variables'
+        inputs:
+          solution: scripts/version/ChangeVersion.proj
```

Comment:
Is it a github highlighting bug? Notepad++ shows no difference between indentation on 258 and 257

Reply:
Probably